packages/frontend/mobile/src/pages/sign-in.tsx

```diff
@@ -1,7 +1,73 @@
-// Default route fallback for mobile
-import { SignIn } from '@affine/core/pages/sign-in';
+import { authAtom } from '@affine/core/atoms';
+import { AuthPanel, type AuthProps } from '@affine/core/components/affine/auth';
+import {
+  RouteLogic,
+  useNavigateHelper,
+} from '@affine/core/hooks/use-navigate-helper';
+import { AuthService } from '@affine/core/modules/cloud';
+import { useLiveData, useService } from '@toeverything/infra';
+import { useAtom } from 'jotai';
+import { useCallback, useEffect } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+
+import { MobileSignInLayout } from '../components/sign-in/layout';
 
 export const Component = () => {
```

Comment:
We have previously agreed that sign-in should use a full-screen modal (implemented in the same way as settings page).

Reply:
I think we still need a standalone sign-in page for mobile since we may only allow logged in user to use the mobile version.
I will update the login modal accordingly.